src/applications/vaos/referral-appointments/ReviewAndConfirm.jsx

```diff
@@ -0,0 +1,231 @@
+  );
+
+  useEffect(
+    () => {
+      if (providerFetchStatus === FETCH_STATUS.notStarted) {
+        dispatch(fetchProviderDetails(currentReferral.providerId));
+      } else if (providerFetchStatus === FETCH_STATUS.succeeded) {
+        setLoading(false);
+        scrollAndFocus('h1');
+      } else if (providerFetchStatus === FETCH_STATUS.failed) {
+        setLoading(false);
+        setFailed(true);
+        scrollAndFocus('h2');
+      }
+    },
+    [currentReferral.providerId, dispatch, providerFetchStatus],
+  );
+
+  useEffect(
+    () => {
+      if (
+        !selectedSlot &&
+        savedSelectedSlot &&
+        providerFetchStatus === FETCH_STATUS.succeeded
+      ) {
+        const savedSlot = getSlotById(provider.slots, savedSelectedSlot);
+        if (!savedSlot) {
+          routeToCCPage(history, 'scheduleReferral');
+        }
+        dispatch(setSelectedSlot(savedSlot.id));
+      }
+    },
+    [
+      dispatch,
+      savedSelectedSlot,
+      provider.slots,
+      history,
+      providerFetchStatus,
+      selectedSlot,
+    ],
+  );
+
+  if (loading) {
+    return (
+      <div className="vads-u-margin-y--8" data-testid="loading">
+        <va-loading-indicator message="Loading schedule referral review..." />
+      </div>
+    );
+  }
+
+  if (failed) {
+    return (
+      <va-alert data-testid="error" status="error">
+        <h2>We’re sorry. We’ve run into a problem</h2>
+        <p>
+          We’re having trouble getting your upcoming appointments. Please try
+          again later.
+        </p>
+      </va-alert>
+    );
+  }
+  const headingStyles =
+    'vads-u-margin--0 vads-u-font-family--sans vads-u-font-weight--bold vads-u-font-size--source-sans-normalized';
+  return (
+    <ReferralLayout hasEyebrow>
+      <div>
+        <h1 data-testid="review-heading">Review your appointment details</h1>
+        <hr className="vads-u-margin-y--2" />
+        <div className=" vads-l-grid-container vads-u-padding--0">
+          <div className="vads-l-row">
+            <div className="vads-l-col">
+              <h2 className={headingStyles}>
+                {`${currentReferral.CategoryOfCare} Provider`}
+              </h2>
+            </div>
+          </div>
+        </div>
+        <p className="vads-u-margin--0">
+          {provider.providerName} <br />
+          {provider.orgName}
+        </p>
+        <ProviderAddress
+          address={provider.orgAddress}
+          phone={provider.orgPhone}
+        />
+        <hr className="vads-u-margin-y--2" />
+        <div className=" vads-l-grid-container vads-u-padding--0">
+          <div className="vads-l-row">
+            <div className="vads-l-col">
+              <h2 className={headingStyles}>Date and time</h2>
+            </div>
+            <div className="vads-l-col vads-u-text-align--right">
+              <va-link
+                href={`/my-health/appointments/schedule-referral/date-time?id=${
```

Comment:
Should this use the `routeTo...` functions rather than an href?

Reply:
the href is there for accessibility, and for people to hover over to see where its going, but does nothing functionally since the onClick is preventing default actions. Its also a required prop for [va-link](https://design.va.gov/storybook/?path=/docs/components-va-link--docs). The routeTo stuff then happens in the onClick

Reply:
I did notice that I wasn't using the right prop for the aria-label though and I adjusted that.

Reply:
> the href is there for accessibility, and for people to hover over to see where its going, but does nothing functionally since the onClick is preventing default actions. Its also a required prop for [va-link](https://design.va.gov/storybook/?path=/docs/components-va-link--docs). The routeTo stuff then happens in the onClick

Ahh gotcha thanks. Not familiar when to use `href` vs a `onClick`